docs/components/concepts/process-instance-creation.md

```diff
@@ -67,20 +73,23 @@ When the client resends the command, it creates a new process instance.
 :::
 
 <details>
-  <summary>Code example</summary>
-  <p>
-Create a process instance and await results:
+   <summary>Create a process instance and await results via Camunda 8 REST API</summary>
+   <p>
 
 ```
-zbctl create instance "order-process" --withResult --variables '{"orderId": "1234"}'
+curl -L 'http://localhost:8080/v2/process-instances' \
+-H 'Content-Type: application/json' \
+-H 'Accept: application/json' \
+-d '{
+  "processDefinitionId": "order-process”,
+  "processDefinitionVersion": 1,
+  "awaitCompletion": true,
+  "variables": { "orderId": "1234" }
+}'
 ```
 
 Response:
 
-:::note
-The variables in the response depend on the process.
-:::
-
 ```
 {
   "processKey": 2251799813685249,
```

Comment:
🔧 We need to adjust the response mapping like we did above

Reply:
Done.